### PR TITLE
Game pad config work

### PR DESCRIPTION
**Pull request by gnrlwart**
Config work for gamepads

### CODE REVIEW

**Review by seblindfors — APPROVED** [review]

File: ConsolePort_Config/Splash/Device.lua [hunk 1]
```diff
@@ -104,6 +117,21 @@ function Devices:UpdateDevices()
 			end
 			device:SetPoint('LEFT', prev, 'RIGHT', FIXED_OFFSET, 0)
 		end
+		if (connectedStyle and (device.Device.LabelStyle == connectedStyle)) then
+			if (device.Device.StyleNameSubStrs) then
```
Comment (seblindfors) [inline ConsolePort_Config/Splash/Device.lua:121]: The array of strings is called `StyleNameMatches` in the PS5 profile

File: ConsolePort/Model/Gamepad/PlayStation/PlayStation5.lua [hunk 2]
```diff
@@ -1,6 +1,8 @@
 select(2, ...).Gamepad:AddGamepad({
 	Name = 'PlayStation 5';
-	Version = 3;
+	LabelStyle = 'Shapes';
+	StyleNameMatches = { [1]='PS5', [2]='DS5', [3]='DualShock 5', [4]='PlayStation 5' };
```
Comment (seblindfors) [inline ConsolePort/Model/Gamepad/PlayStation/PlayStation5.lua:4]: This array of strings is referred to as `StyleNameSubStrs` in [Device.lua](#diff-2359d1ce6afc7204c0d039c8ad4886a2853e5caf25b6bfd3d99db55ab03fdaf2)

**Review by seblindfors — APPROVED** [review]

File: ConsolePort/Model/Gamepad/PlayStation/PlayStation5.lua (hunk 2 above)
Comment (gnrlwart) [reply ConsolePort/Model/Gamepad/PlayStation/PlayStation5.lua:4]: Ack, that's what I get for "cleaning up" after removing my debug prints.

Comment (gnrlwart) [reply ConsolePort/Model/Gamepad/PlayStation/PlayStation5.lua:4]: Sigh... they're both StyleNameSubStrs in VS Code, but not GitHub. I'll figure out where things went off the rails.

Comment (gnrlwart) [reply ConsolePort/Model/Gamepad/PlayStation/PlayStation5.lua:4]: TIL VS Code's Save-All hotkey is different from VS's, and it apparently doesn't ask to save changes when you close it. :-|

**Pull request merged**